src/lib.rs

```diff
@@ -24,6 +24,20 @@ pub enum Error<E> {
 /// I²C address
 pub const ADDRESS: u8 = 0x40;
 
+pub struct MilliCelsius(i32);
+pub struct RHPerCentMille(i32);
+
+impl MilliCelsius {
+    pub fn as_millicelsius(&self) -> i32 { self.0 }
+    pub fn as_celsius(&self) -> f32 { self.0 as f32 / 1000. }
+}
+
+impl RHPerCentMille {
+    pub fn as_percentmille(&self) -> i32 { self.0 }
```

Comment:
Is `percentmille` a real word? It basically means "one thousandth of one hundredth".

It seems that `as_permille` would be the proper name: https://en.wikipedia.org/wiki/Per_mille

Reply:
Thanks for your comments - now it's my turn to be a bit (ahem) slow to reply...

> I'd prefer newtypes that are named after their meaning, not after their internal units

OK, I'll update the PR accordingly.

> It seems that `as_permille` would be the proper name: https://en.wikipedia.org/wiki/Per_mille

My reading of Wikipedia, is:

| Name | Denominator |
| ---- | -------- |
| Percent (%) | 10² |
| Per mille (‰) | 10³ |
| Per cent mille (pcm) | 10⁵ |
| Parts per million (ppm) | 10⁶ |

https://en.wikipedia.org/wiki/Per_cent_mille

... and the code currently calculates integers which are:
- Temperature in units of thousandths of a degree Celsius
- Relative Humidity in units of thousandths of a percent RH (pcm)

... so I think pcm is the correct name for the return type of your optimised integer RH calc at the moment.

Alternatively It might be less weird to alter the function to calculate ppm instead (since ppm is a bit less "niche" than pcm), and that'll still easily fit into the 32 bit integer return type.  Any thoughts?

Reply:
Ah, that makes sense. "per mille" is 1/1000 and "per cent mille" is one percent of a "per mille".